### DISCUSSION TITLE
Is it alright if I update my associated email address for the competition? If so, how do I that?

### DISCUSSION THREAD

**Discussion by TechEvolvace** [SimDynamX/SpaceTeamsPRO#52] [2024 Lunar SAR Competition Q&A] [answered]
# Inquiry
Hi, I'm Phuc! I was wondering if it's possible to update our associated email address for this competition. I received the email that gets sent out to all members of qualified teams, so I know that I'm in the competition now. 
![Welcome Email](https://github.com/user-attachments/assets/14f90b54-fe0f-4773-ba40-34b351228c88)

I just want to update my email address from ams.dinhphuc@gmail.com to EvolvaceTechnical.PhucDinh@proton.me. It's to help organize my inbox better if I know where the emails from this competition gets sent to. I'm not sure, but do I update my email address through submitting another team member sign up form? 
![Email Change Inquiry Part 1](https://github.com/user-attachments/assets/1929cb8b-c144-4c37-ab71-3bcbbcd86eff)

![Email Change Inquiry Part 2](https://github.com/user-attachments/assets/ca5e53e4-eec3-43c8-951b-5026ecaac036)

Based on the instructions from the Getting Started Guide, it looks that where you keep track of each member in this competition and their email address. Submitting another team member sign up form to update my associated email address might be an option, but I don't mind if you update my associated email address for this competition manually.

**Comment by connorjak** [comment]
We maintain the email list manually; I'll forward this to the person who sends out the annoucements

**Reply by TechEvolvace** [reply]
@connorjak Alright, let me know when that person responds.

**Reply by connorjak** [reply, marked as answer]
Should be addressed by now, reopen the discussion and message if not.
